examples/GainPlugin/Source/jsui/package.json

```diff
@@ -3,43 +3,21 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@babel/runtime-corejs3": "^7.10.3",
```

Comment:
This may be nonsense but I had a set of configurations and dependency versions working well in my current project so I copied some stuff over for good measure.